.contrib/Parser/DATAS/06 - Expansion Features/05 Island Expeditions/Quests.lua

```diff
@@ -17,6 +17,37 @@ _.ExpansionFeatures =
 			}),
 		}),
 		n(QUESTS, {
+			-- These quests need to be done once per account to unlock Island Expeditions. After completing, each character will have completed the flag quest 51994.
```

Comment:
do these quests become unavailable on alts? if yes it would make sense to use `altQuests`, otherwise it would just hide this short questline on alts even if it's not completed?

Reply:
Correct.
Every alt I create (Horde or Alliance, even on an alternative WoW account under the same Battle.net (including trial accounts)), quest 51994 is instantly marked as Completed, like on this level 1 unactivated WoW account in my Battle.net:
![image](https://user-images.githubusercontent.com/4090858/87552771-663d6780-c6b2-11ea-8c93-1bd4a43b8964.png)

Island Expeditions are available even without doing the questline aswell.

Reply:
Okay, if the questline is only doable once, it's fine, just wanted to make sure that we don't mark available quests as completed with `altQuests`.